### ISSUE TITLE
Provide public getters for properties of AbstractConnectionResolver class

### ISSUE THREAD

**Issue by jasonbahl** [wp-graphql/wp-graphql#958]
The AbstractConnectionResolver provides central filters that apply to _all_ connections extending the class, making it possible for plugins / themes to filter various functionality of Connections in a central way. 

The instance of the Connection is passed through to the filters, but many of the properties of the class aren't publicly exposed, making it difficult for extending by checking the values of the properties.

Take this filter for example, where we want to check if the connection was passed specific arguments, and if it was, apply it the the query_args that will be used by WP_Query (or other query class):

```
add_filter( 'graphql_connection_query_args', function( $query_args, $connection ) {

  if ( isset( $connection->args['where']['someCustomInput'] ) ) {
    $query_args['something_custom'] = $connection->args['where']['someCustomInput'];
  }

  return $query_args;

} );
```

The `$connection->args` are not public, so this won't work. We will see a 

Exposing public getters for the properties that should be available will allow users to make use of these filters more appropriately. 

For example, this change (using ->getArgs()) will work properly if that's a public method: 

```
add_filter( 'graphql_connection_query_args', function( $query_args, $connection ) {

  if ( isset( $connection->getArgs()['where']['someCustomInput'] ) ) {
    $query_args['something_custom'] = $connection->getArgs()['where']['someCustomInput'];
  }

  return $query_args;

} );
```

**Comment by jasonbahl** [comment]
closed by #959

**Comment by jasonbahl** [comment]
Thanks @CodeProKid
